inc/az_ulib_descriptor_api.h

```diff
@@ -150,18 +147,16 @@ typedef struct az_ulib_interface_descriptor_tag
  *                            cannot be `NULL` and shall be allocated in a way that it stays valid
  *                            until the interface is unpublished at some (potentially) unknown time
  *                            in the future.
- * @param[in]   command       The function pointer to #az_ulib_capability_command with the
+ * @param[in]   method        The function pointer to #az_ulib_capability_command with the
```

Comment:
I'll rename to command_target to avoid issues in the initialization. 

---
In reply to: [605802314](https://github.com/Azure/azure-ulib-c/pull/65#discussion_r605802314) [](ancestors = 605802314)